bundles/http/Citrix/Covid-19-Self-Certify/metadata.json

```diff
@@ -0,0 +1,30 @@
+{
+  "type" : "HTTP",
+  "vendor" : "Citrix",
+  "id" : "1c91bed3-4e8a-4979-b442-c2ddba97c7f7",
+  "trackingUuid" : "d07b643f-ef4e-40b8-a739-96f4d18ea603",
+  "title" : "Covid-19 Self Certify",
+  "description" : "Deploy the \"Covid-19 Self Certify\" micro-app to easily submit a self-certification process using Citrix Workspace and Podio.",
+  "iconUrl" : "https://iws-prod-global-customicons-cdn-endpoint.azureedge.net/image/pjBX0919Ni9PyI15Wx2qvX1TAt10pvpt.png",
+  "masVersion" : "1.108.1.a664ca4c65937ceaa41e9ad1acad66eec5d4a3e2",
+  "categories" : [ "WEB_SERVICES" ],
+  "created" : "2021-08-16T08:27:07",
+  "supportsOAuthForActions" : false,
+  "i18nLanguages" : [ "en" ],
+  "apps" : [ {
+    "uuid" : "e0fc92f2-8671-4725-91e1-05fa60af91be",
+    "trackingUuid" : "deec9060-18e0-4bc9-9b82-3c38af0cc77d",
+    "title" : "Self Certify",
+    "numberOfEvents" : 4,
+    "action" : true
+  } ],
+  "vaResolvers" : [ ],
+  "tags" : [ ],
```

Comment:
Please add the helpLink for documentation before I can merge. Thanks!

Reply:
Added the documentation link in metadata.json file.